statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -697,6 +952,48 @@ def hqic(self):
         # return -2*self.llf + 2*np.log(np.log(self.nobs))*self.params.shape[0]
         return hqic(self.llf, self.nobs, self.params.shape[0])
 
+    @property
+    def cov_type(self):
+        return self._cov_type
+    @cov_type.setter
+    def cov_type(self, value):
```

Comment:
The attribute set on line 808 uses the property setter. But looking at this again, I think using this as a property may be a problem when using the wrapper class.